views/accessibility-statement.html

```diff
@@ -0,0 +1,112 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Accessibility statement – {{ SERVICE_NAME }} - GOV.UK
```

Comment:
Small thing but are the hyphens the correct ones? I remember we had to use '-' to avoid issues

Reply:
It is not an issue from a technical point of view, the browser will render correctly the hyphen (-) or dashes (— or –). For consistency I will update it to match other pages